cli/ceph/orch/orch.py

```diff
@@ -0,0 +1,133 @@
+from ceph.utils import get_nodes_by_ids
+from cli import Cli
+from cli.utilities.core_utils import config_dict_to_string
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class OrchApplyServiceFailure(Exception):
+    pass
+
+
+class Orch(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(Orch, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} orch"
+
+    def ls(self, **kw):
+        """
+        Lists the status of the given services running in the Ceph cluster.
+
+        Args:
+            kw (Dict): Key/value pairs that needs to be provided to the installer.
+            Example:
+                Supported keys:
+                    service_type(str): type of service(mon, osd, mgr, mds, rgw)(Optional).
+                    service_name(str): name of host(Optional).
+                    export(bool): Whether to export the service specifications knows to
+                                  the orchestrator(Optional).
+                    format(str): the type to be formatted(yaml)(Optional).
+                    refresh(bool): Whether to refresh or not(Optional).
+        """
+        args = ""
+        for k, v in kw.items():
+            if v is True:
+                args = args + f" --{k}"
+            else:
+                args = args + f" --{k} {v}"
+        cmd = f"{self.base_cmd} ls {args}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def apply(self, cluster=None, **kw):
```

Comment:
Don't pass cluster here. Add below method in `cli/utilities/utils.py` to get node object by it's id -
```
get_node_by_id(nodes. id)
```